packages/use-presence/test/usePresenceSwitch.test.tsx

```diff
@@ -64,14 +58,16 @@ it("can animate the exit and re-entrance of a component that has changed it's re
 });
 
 it("can animate the initial entrance and exit of a component based on it's rendered data", async () => {
-  const {getByTestId, getByText, rerender} = render(<Expander text={null} />);
+  const {getByTestId, getByText, rerender} = render(
+    <Expander text={undefined} />
+  );
   getByTestId('none');
   getByText('Nothing mounted');
   rerender(<Expander text="initial value" />);
   getByTestId('isAnimating, isEntering, isMounted');
   getByText('initial value');
   await waitFor(() => getByTestId('isVisible, isMounted'));
-  rerender(<Expander text={null} />);
+  rerender(<Expander text={undefined} />);
```

Comment:
I originally did this but the tests complained in a warning and told me to use `null`